### DISCUSSION TITLE
zh/guide/advanced/ipa

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#18] [Announcements]
# zh/guide/advanced/ipa

AList v3的文档

https://alist.nn.ci/zh/guide/advanced/ipa.html

**Comment by jinchunyou** [comment]
我想请问一下，V2版本支持吗

**Reply by xhofe** [reply]
不支持

**Comment by uinioyun** [comment]
从App Store抓包ipa后，放到v3里面，就可以直接下载安装了？不用重新签名了吗？

**Comment by yangyzp** [comment]
使用  ipa-renamer 重命名后，通过alist v3还是无法安装，虽然桌面上有cloud那个开始安装应用的图标了，但是无反应，点击那个图标，提示“无法安装”

**Reply by ZJC1** [reply]
一样，你解决了？

**Reply by chinnsenn** [reply]
我也一样，提示无法安装

**Comment by at86397388** [comment]
ipa文件在哪里下载呢

**Comment by insmine** [comment]
hi，iOS8以下的iOS设备无法使用这个功能，能否在下一次更新中加入“识别出iOS环境直接使用itms-services协议在线安装IPA”的功能，谢谢

**Comment by a5478041** [comment]
修改了文件名还是不能在线安装怎么解决

**Reply by ios33** [reply]
换工具

**Comment by Charlotte3240** [comment]
我想问一下这个功能是需要自己先进行签名再上传吗

**Comment by Contribuv** [comment]
成功了，但是请优化前端显示，
比如 微信@com.tencent.xin.ipa ，在前端去掉 @之后的显示，不去掉整个页面，畸形难看，排版差。

**Comment by Animism001** [comment]
iOS16点击安装，提示网站的证书无效，有人知道怎么解决吗？十分感谢

**Reply by ilson-alex12** [reply]
我也一样，提示证书无效，怎么添加有效的证书啊。

**Reply by FopFat** [reply]
请问在哪里下载？

**Reply by FopFat** [reply]
额。找到了。

**Comment by YoisakiKnd** [comment]
未测试iOS16，但是iOS15会出现无法验证完全性的问题，设备管理中没有需要信任的证书，请问可以怎样解决

**Comment by ghost** [comment]
ios16完美使用, 楼上不能用的  提示一下 ipa安装 需要https和 有效的证书.

**Reply by lizhiwei8061** [reply]
求助：搭建的webdav,ios无法访问，请求报错：Synchronizer: "TypeError: Network request failedTypeError: Network request failed

**Reply by ghost** [reply]
> 求助：搭建的webdav,ios无法访问，请求报错：Synchronizer: "TypeError: Network request failedTypeError: Network request failed

alist正常部署之后 用nginx反代 使用https 就可以了

**Reply by lizhiwei8061** [reply]
那段时间我启用alist ssl配置也没成功呢

**Reply by lizhiwei8061** [reply]
请问有没有教程，我用这个反向代理，倒是op进不去了
location / {
 proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_set_header X-Forwarded-Proto $scheme;
  proxy_set_header Host $http_host;
 proxy_set_header X-Real-IP $remote_addr;
  proxy_set_header Range $http_range;
  proxy_set_header If-Range $http_if_range;
  proxy_redirect off;
  proxy_pass http://127.0.0.1:5244;
  the max size of file to upload
  client_max_body_size 20000m;
 }

**Comment by lizhiwei8061** [comment]
求助：搭建的webdav,ios无法访问，请求报错：Synchronizer: "TypeError: Network request failedTypeError: Network request failed

**Comment by Animism001** [comment]
应该是ios端的软件设置错误，如果是infuse，那所有的的选项都要填写正确，位置192.168.x.x，不要包含http高级选项里路径填写dav,端口5244。

**Reply by lizhiwei8061** [reply]
地址是相同的 android pc端访问都正常，就是ios手机手机访问不行，Google有说ios手机安全问题 但是没找到解决方案，所以来问问

**Comment by Animism001** [comment]
把你用的软件以及填写的选项发出来别人才知道哪里出问题了，你这样问谁知道是哪里出问题了

**Reply by lizhiwei8061** [reply]
使用的软件是Joplin, 以下是完整报错：Synchronizer:"TypeError: Network request failedypeError: Network request failedat anonymous (/private/var/containers/Bundle/Application/B9B738F3D639-4741-846A-3A9044F38CEC/Joplin.app/mainjsbundle:94:8201)at apply (native)
at anonymous (/private/var/containers/Bundle/Application/B9B738F3D639-4741-846A-3A9044F38CEC/Joplin.app/main.jsbundle:72:1247)at k (/private/var/containers/Bundle/Application/B9B738F3-D639-4741-846A-3A9044F38CEC/Joplin.app/mainjsbundle:72:498)at callTimers (/private/var/containers/Bundle/Application/B9B738F3D639-4741-846A-3A9044F38CEC/Joplin.app/mainjsbundle:72:2647)
at apply (native)at value (/private/var/containers/Bundle/Application/B9B738F3D639-4741-846A-3A9044F38CEC/Joplin.app/mainjsbundle:35:3798)at anonymous (/private/var/containers/Bundle/Application/B9B738F3D639-4741-846A-3A9044F38CEC/Joplin.app/main.jsbundle:35:674)at value (/private/var/containers/Bundle/Application/B9B738F3-D639-4741-846A-3A9044F38CEC/Joplin.app/main.jsbundle:35:2504)at value (/private/var/containers/Bundle/Application/B9B738F3-D639-4741-846A-3A9044F38CEC/Joplin.app/main.jsbundle:35:646)03-27T10:32:52:

**Comment by Animism001** [comment]
下载您出说的软件试了一下，没有出现问题。
WebDAV URL 填写的是http://192.168.x.x:5244/dav/xxx/xxx。你看看是不是你填写的格式有问题。如果你的alist配置文件改过，那就按照你改过的内容来填写。

**Reply by lizhiwei8061** [reply]
这个地址的填写用ip地址确实能测试成功，但是同步数据的时候还是报错，如果使用域名测试就报上面的错误

**Reply by lizhiwei8061** [reply]
我今天又测试了一下，用url上填写IP同步数据成功了，但是使用域名依旧是不可以

**Comment by lizhiwei8061** [comment]
求助：搭建的webdav,苹果手机同步异常，其他终端访问都正常，使用的app是Joplin,报错是 Synchronizer:"TypeError: Network request failedypeError: Network request failed，并且Alist配置中启用了ssl

**Comment by ht9z2** [comment]
ipa安装的页面 没有复制链接

**Comment by a49942245a** [comment]
安装ipa的时候桌面 app呈现灰色的“等待中”图标，不显示进度条，必须下载完才会走进度条，此问题能解决吗？亲测蓝奏云下载ipa有正常进度条显示
